frontend/src/core/codemirror/language/__tests__/sql.test.ts

```diff
@@ -735,16 +735,130 @@ describe("tablesCompletionSource", () => {
     expect(completionSource?.schema).toMatchInlineSnapshot(`
       {
         "public": {
-          "orders": [
-            "order_id",
-            "user_id",
-            "total",
-          ],
-          "users": [
-            "id",
-            "name",
-            "email",
-          ],
+          "children": {
```

Comment:
these snapshots changed for two reason:

1. instead of just labels, it now includes a custom Completion with a `info` render
2. We also include default databases at the top (previously did not). that can be see in this commit: https://github.com/marimo-team/marimo/pull/6502/commits/780202d8c95d24448693121ee85385d1557abea3#diff-f79af3635ef377d486695e4101902aad0982c208b769b3851c9086e694f722f3